Add Arcaplanet

Nicos18 opened the pull request.

Nicos18 committed:
Update retail.yml

Nicos18 committed:
Add files via upload

mxxcon reviewed (changes requested):
Their official logo ![arcaplanet](https://user-images.githubusercontent.com/847140/30920030-fa424a12-a370-11e7-9614-ef9473a97975.png)

Nicos18 committed:
Add files via upload

jamcat22 committed:
Arcaplanet is HTTPS and fix Twitter capitalization

Arcaplanet's website now redirects to HTTPS, and their Twitter username is now capitalized.
